pages/[user]/comments.js

```diff
@@ -0,0 +1,144 @@
+import {shallowEqual, useSelector} from 'react-redux';
```

Comment:
I think we should introduce some enforcing of code formatting, using ESLint or Prettier. But until then, please fix the spaces here (the same for `likes.js`).